src/solvers/dgmulti/flux_differencing.jl

```diff
@@ -0,0 +1,199 @@
+"""
+    hadamard_sum_A_transposed!(du, ATr, volume_flux, u, skip_index=(i,j)->false)
+
+Computes the flux difference ∑_j A[i, j] * f(u_i, u_j) and accumulates the result into `du`.
+- `du`, `u` are vectors
+- `ATr` is the transpose of the flux differencing matrix `A`. The transpose is used for
+  faster traversal since matrices are column major in Julia.
+"""
+@inline function hadamard_sum_A_transposed!(du, ATr, volume_flux, u, skip_index=(i,j)->false)
```

Comment:
```suggestion
@inline function hadamard_sum_A_transposed!(du, ATr, volume_flux, u, skip_index)
```
The method without `skip_index` is below. Including this one here yields
```julia
julia> using Trixi
[ Info: Precompiling Trixi [a7f1ee26-1774-49b1-8366-f1abc58fbfcb]
WARNING: Method definition hadamard_sum_A_transposed!(Any, Any, Any, Any) in module Trixi at .../Trixi/src/solvers/dgmulti/flux_differencing.jl:9 overwritten at ...Trixi/src/solvers/dgmulti/flux_differencing.jl:25.
  ** incremental compilation may be fatally broken for this module **
```

Reply:
Thanks for fixing that!